_data/glossary/statefulset.yml

```diff
@@ -0,0 +1,18 @@
+id: statefulset
+name: StatefulSet
+formerly:
+- PetSet
+related:
+- Deployment
+- Pod
+tags:
+- Storage
+- Workload
+- API Object
+short-description: |
+  Manages the deployment and scaling of a set of replica Pods, *and provides guarantees about ordering*. It does so by maintaining a *unique*, sticky identity for each of its Pods.
```

Comment:
@chenopis @Bradamant3 

Yeah, see:
<img width="1058" alt="screen shot 2017-08-23 at 10 31 42 am" src="https://user-images.githubusercontent.com/25401650/29629884-35bd98e6-87ef-11e7-8270-e3e194234593.png">

^I wondering if the sentence-fragment structure of `short-description` seems a bit out of place in the header section of a Concept page (in addition to the singular/plural agreement). It could be the way that I phrased this example though, if so, open to suggestions.